110_Multi_Field_Search/35_Entity_search.asciidoc

```diff
@@ -73,19 +62,12 @@ combine the scores of all matching fields:
 }
 --------------------------------------------------
 
-==== Problems with the most_fields Approach
+==== most_fields 方式的问题
 
-The `most_fields` approach to entity search has some problems that are not
-immediately obvious:
-
-* It is designed to find the most fields matching _any_ words, rather than to
-  find the most matching words _across all fields_.
-
-* It can't use the `operator` or `minimum_should_match` parameters
-  to reduce the long tail of less-relevant results.
-
-* Term frequencies are different in each field and could interfere with each
-  other to produce badly ordered results.
+用 `most_fields` 这种方式搜索也存在某些问题，这些问题并不会马上显现：
 
+* 它是为多数字段匹配 _任意_ 词设计的，而不是在 _所有字段_ 中找到最匹配的。
 
+* 它不能使用 `operator` 或 `minimum_should_match` 参数来减少次相关结果中的长尾。
```

Comment:
It can't use the `operator` or `minimum_should_match` parameters
 -  to reduce the long tail of less-relevant results.
这里可不可以这样翻译：
它不能使用 `operator` 或 `minimum_should_match` 参数来降低不相关结果造成的长尾效应